### ISSUE TITLE
Support for mongoid 5

### ISSUE THREAD

**Issue by sivagollapalli** [stevebartholomew/newrelic_moped#36]
As mongoid driver changed from moped to mongo. Would there be any plan to support mongoid 5?

**Comment by tonyta** [comment]
@sivagollapalli it looks like `newrelic_rpm` will have built in support for Mongo Ruby Driver in Mongoid 5

**Comment by stevebartholomew** [comment]
Yes, I believe this is the case.  I need to make this gem/repo deprecated in favour of the newer, built in functionality.

**Comment by rachelleahklein** [comment]
Hi @stevebartholomew. On behalf of the New Relic Ruby agent engineers, thank you for creating this gem to help our customers with Mongo tracing before we had sufficient support in the agent itself.

If it's indeed the case that the agent's built-in functionality is now preferable to using this gem, would it be possible to clearly note that (e.g. in your README) and archive this repo in order to prevent any confusion?

And if you have anything to add to our agent's instrumentation, we would welcome a pull request from you.

Again, thank you for all your work.

**Comment by stevebartholomew** [comment]
@rachelleahklein thanks for your kind words. 

I have added a message to the readme and linked directly to the mongodb repo - let me know if you'd like it to link elsewhere.
